### DISCUSSION TITLE
SECTION-68 テストファイルのファイル名について

### DISCUSSION THREAD

**Discussion by fuuukeee3** [budougumi0617/go_todo_app#51] [Q&A]
- bad_req.json.golden
- bad_req_rsp.json.golden
- ok_req.json.golden
- ok_rsp.json.golden
上記のようにテストファイルを準備しますが、`bad_req_rsp.json.golden`は、okケースのテストファイルと名称を統一するなら、` bad_rsp.json.golden`となるのではないでしょうか。

**Comment by budougumi0617** [comment]
ご助言ありがとうございます！確かにファイル名が雑でしたので正誤表とサンプルコードを修正いたしました。

https://github.com/budougumi0617/go_todo_app/releases/tag/v1.0.7
